src/screens/Register/Register.jsx

```diff
@@ -333,6 +357,46 @@ const Register = ({ image }) => {
                 show={!baseURL}
               />
             )}
+
+            <div
+              className={`checkbox-wrapper ${!isPrivacyAvailable && "hidden"}`}
+            >
+              <input
+                type="checkbox"
+                name="privacy"
+                onChange={handleReadPrivacy}
+              />
+              <label className="label">
+                {privacyTextBefore}
+                <LinkCreator
+                  path="/privacy-policy"
+                  setValid={setIsPrivacyAvailable}
+                >
+                  {privacyTextLink}
+                </LinkCreator>
+                {privacyTextAfter && <p>{privacyTextAfter}</p>}
+              </label>
+            </div>
+
+            <div
+              className={`checkbox-wrapper ${!isTermsAvailable && "hidden"}`}
+            >
+              <input
+                type="checkbox"
+                name="terms"
+                onChange={handleAcceptTerms}
+              />
+              <label className="label">
+                {termsTextBefore}
+                <LinkCreator
+                  path="/terms-and-conditions"
+                  setValid={setIsTermsAvailable}
+                >
+                  {termsTextLink}
+                </LinkCreator>
+                <p>{termsTextAfter}</p>
```

Comment:
above you added a check. Is that check necessary?

Reply:
no not necessary. It renders just an empty p-tag